securesystemslib/keys.py

```diff
@@ -522,6 +522,10 @@ def format_metadata_to_key(key_metadata, default_keyid=None):
       provided, the keyid will be calculated by _get_keyid using the default
       hash algorithm. If provided, the default keyid can be any string.
 
+    hash_algorithms:
+      An optional parameter used to provide custom hash algorithms when
```

Comment:
I may be over-reading this, but I'm finding the term "custom" to be problematic. It's not a custom hash algorithm, it's choosing which of the supported hash algorithms you want to use.
```suggestion
      An optional list of hash algorithms to use when
```

Reply:
Yes, not a bad idea and this will provide information about the type of that argument which is good.

I add and squashed my previous commits.